src/io/isam.md

```diff
@@ -41,190 +41,191 @@ READ(channel, data_area, key_spec[, KEYNUM:key_num][, LOCK:lock_spec]
+- **WAIT**: Optional. Specifies how long the program should wait for a record to become available if it's locked. This is more efficient than calling READ in a loop with a SLEEP statement because the call will be blocked until the record is available rather than waiting the entire sleep duration and then checking again.
 
-**Functional Insights**:
+**Functional insights**:
 - The `LOCK` qualifier specifies whether a manual lock is applied to the record. This feature is critical in concurrent environments where data integrity is paramount.
-- The `MATCH` qualifier defines how a record is located. This determines the behavior when an exact match isnt found. Due to historic reasons, the default behavior is `Q_GEQ` but this is likely to lead to an unpleasant surprise for someone writing new code. The following options are available:
-  - **Q_GEQ (0)**: Finds a record with a key value greater than or equal to the specified key. If an exact match isn't found, it reads the next higher record, potentially raising a `$ERR_KEYNOT` error if no exact match is found, or an “End of file” error if no higher record exists.
-  - **Q_EQ (1)**: Searches for a record that exactly matches the specified key value. If there's no exact match, it triggers a `$ERR_RNF` error.
-  - **Q_GTR (2)**: Looks for a record with a key value greater than the specified key. It only raises an `$ERR_EOF` error if it reaches the end of the file without finding a matching record.
-  - **Q_RFA (3)**: Locates a record using the Record File Address (RFA) specified in the RFA qualifier.
-  - **Q_GEN (4)**: Finds a record whose key value is equal to or the next in sequence after the specified key. An `$ERR_EOF` error occurs if no such records are found.
-  - **Q_SEQ (5)**: Retrieves the next sequential record after the current one, regardless of the key specification. It raises an “End of file” error only if it reaches the end of the file.
+- The `MATCH` qualifier defines how a record is located. This determines the behavior when an exact match isn't found. For historical reasons, the default behavior is `Q_GEQ`, but that is likely to lead to an unpleasant surprise for someone writing new code. The following options are available:
+  - **Q_GEQ (0)**: Finds a record with a key value greater than or equal to the specified key. If an exact match isn't found, it reads the next higher record, potentially raising a \$ERR_KEYNOT error if no exact match is found or an “End of file” error if no higher record exists.<!--Is there a reason we use the mnemonic for KEYNOT but the text for EOF? I think we should use the mnemonic for both-->
```

Comment:
I don't really like the mnemonics conversationally where there is a general concept from software engineering. ERR_KEYNOT is special and has somewhat unique meaning to environments like ours, while end of file is generalized to anyone writing software that interacts with files.